app.js

```diff
@@ -449,6 +449,16 @@ app.get("/terms", routes.angular);
 app.get("/privacy", routes.angular);
 app.get("/languages", routes.page("languages"));
 
+// goggles onboard, with special image routing for lowest-threshold onboarding
+app.get("/goggles", routes.angular);
+app.get("/goggles/install", routes.angular);
```

Comment:
would rather keep this explicit, because someone down the line is going to search for "/install" and it's nicer to them to have a search hit instead of a mystery star =)